src/app/core/actions/addition/SplitWireAction.ts

```diff
@@ -8,13 +8,22 @@ import {ConnectionAction, DisconnectAction} from "./ConnectionAction";
 import {PlaceAction, DeleteAction} from "./PlaceAction";
 
 
+
 export function CreateSplitWireAction(designer: CircuitDesigner, w: Wire, port: Node): GroupAction {
     const action = new GroupAction();
-
+    
     action.add(new DisconnectAction(designer, w).execute());
     action.add(new PlaceAction(designer, port).execute());
-    action.add(new ConnectionAction(designer, w.getP1(), port.getP1()).execute());
-    action.add(new ConnectionAction(designer, port.getP2(), w.getP2()).execute());
+    //creates and saves new ConnectionAction to a var and then executes the action.
```

Comment:
nit: put extra line above this
nit: space after `//`
nit: capitalize first letter of comment